public/js/thirdpartyservices.js

```diff
@@ -173,5 +173,15 @@ var thirdPartyServicesJsonArray = [
       privacy: "https://expo.io/privacy",
       terms: "https://expo.io/terms"
     }
+  },
+  {
+    name: "Sentry",
+    model: "sentry",
+    expo: false,
+    logo: "images/sentry.svg",
```

Comment:
I think you meant `sentry.png` , since that is the file you added above :thinking:

Reply:
Uh, of course!

Reply:
I will fix that. No worries.

Reply:
Thank you!